manual/cops_rails.md

```diff
@@ -709,6 +709,18 @@ Enabled | No
 
 This cop checks for the use of output calls like puts and print
 
+### Example
+
+```ruby
+# bad
+puts 'A debug message'
+pp 'A debug message'
+print 'A debug message'
+
+# good
+logger.debug 'A debug message'
```

Comment:
`logger` is not always available. For example, we can't use `logger` in model directly. I think `Rails.logger.debug` is better for the example, what do you think?

Reply:
Agreed